src/ert/scheduler/lsf_driver.py

```diff
@@ -265,6 +267,8 @@ async def submit(
         self._iens2jobid[iens] = job_id
 
     async def kill(self, iens: int) -> None:
+        if iens in self._finished_iens:
+            return
```

Comment:
I think we are just covering an underlying problem with this, and should rather ensure we don't call `kill()` when we should not.

Reply:
Maybe we can move the `if iens in self._finished:...` check to before the place `driver.kill(...)` is called.

Reply:
Should the `finished` state be moved to the individual jobs (job.py) instead? That way we can check directly before calling `driver.kill(...)`

Reply:
https://github.com/equinor/ert/pull/7660 is one possible solution that will fix this.

Reply:
@berland Great! Can I reassign you to this issue?